controller/controller.go

```diff
@@ -43,7 +43,16 @@ type Config struct {
 	// and different resources can be executed depending on the runtime object
 	// being reconciled.
 	ResourceRouter *ResourceRouter
-	K8sClient      kubernetes.Interface
+	// RESTClient needs to be a rest.interface which is configured with a serializer
+	// capable of serializing and deserializing the obect which is watched by the
+	// informer.
+	//
+	// For standard k8s object this is going to be e.g.
+	//   k8sClient.CoreV1().RESTClient()
+	//
+	// For CRs of giantswarm this is going to be e.g.
+	//   g8sClient.CoreV1alpha1().RESTClient()
```

Comment:
Format nazi here. I would love to see it like this. The 4 spaces indentation makes it prettier in godoc.org AFAIK. And it is more blocky for my weak human eyes. <3

```
// For CRs of giantswarm this is going to be e.g.
// 
//     g8sClient.CoreV1alpha1().RESTClient()
//
```

Reply:
I used 4 spaces now and added some more new lines, I think it's blocky now :D